reaper_test.go

```diff
@@ -55,11 +54,9 @@ func createContainerRequest(customize func(ContainerRequest) ContainerRequest) C
 		ReaperImage:  "reaperImage",
 		ExposedPorts: []string{"8080/tcp"},
 		Labels: map[string]string{
-			TestcontainerLabel:                "true",
-			TestcontainerLabelIsReaper:        "true",
-			testcontainersdocker.LabelReaper:  "true",
-			testcontainersdocker.LabelLang:    "go",
-			testcontainersdocker.LabelVersion: internal.Version,
+			TestcontainerLabel:               "true",
+			TestcontainerLabelIsReaper:       "true",
```

Comment:
I think we should deprecate this variable, as there exists an internal one with the same key: `testcontainersdocker.LabelReaper`